src/api/index.js

```diff
@@ -41,52 +39,7 @@ function validFetch(path, options, requiresToken) {
   return fetch(path, fetchOptions).then(res => validateResponse(res));
 }
 
-function fetchDelete(path) {
-  return validFetch(path, { method: "delete" }, false);
-}
-
-function fetchJson(path, options = {}) {
-  return validFetch(path, options, false);
-}
-
-function postPutJson(path, body, method) {
-  return fetchJson(path, { method, body: JSON.stringify(body) });
-}
-
 //Base
-export function me() {
```

Comment:
Also remove these API calls from the rest of the codebase (see the warnings when starting the server)